hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/metrics/HybridSubscriptionMetricsRepository.java

```diff
@@ -43,9 +51,17 @@ public HybridSubscriptionMetricsRepository(GraphiteClient graphiteClient, Metric
     @Override
     public SubscriptionMetrics loadMetrics(TopicName topicName, String subscriptionName) {
         String rateMetric = metricPath(topicName, subscriptionName);
+        String timeouts = metricPathTimeouts(topicName, subscriptionName);
+        String otherErrors = metricPathOtherErrors(topicName, subscriptionName);
 
         GraphiteMetrics metrics = graphiteClient.readMetrics(rateMetric);
 
+        List<String> httpStatusCodes = allHttpStatusCodeClasses()
+                .map(code -> metricPathHttpStatuses(topicName, subscriptionName, code))
+                .map(m -> graphiteClient.readMetrics(m)
```

Comment:
You have added 7 additional request to graphite. However GraphiteClient allows you to read different metrics in one request.